### ISSUE TITLE
Parcel is not updating when custom props are changed in the host app

### ISSUE THREAD

**Issue by victorlevasseur** [single-spa/single-spa-angular#140]
Hi,

I just began to use single-spa with single-spa-angular to try the Parcel feature. It seems that the Angular parcel component is ready to allow custom props updates (see https://github.com/single-spa/single-spa-angular/blob/master/src/browser-lib/parcel-lib/parcel.component.ts#L66) but the singleSpaAngular function to create a Parcel config is not providing an update function (see https://github.com/single-spa/single-spa-angular/blob/master/src/browser-lib/single-spa-angular.ts#L37)

Should something be done to fix that? I would happily contribute to fix this as I'm currently facing this limitation.

**Comment by victorlevasseur** [comment]
Here is what I added to make the update work without touching the single-spa-angular code base:
```
const LogoutButtonParcelInternal = singleSpaAngular({
  bootstrapFunction: (props) => {
    singleSpaPropsSubject.next(props);
    return platformBrowserDynamic().bootstrapModule(AppModule);
  },
  template: '<lbp-logout-button />',
  NgZone
});
LogoutButtonParcelInternal.update = (props) => { // Needed so that we receive the new props if their are updated.
  singleSpaPropsSubject.next(props);
  return Promise.resolve();
};

export const LogoutButtonParcel = LogoutButtonParcelInternal;
```

Maybe we can add an updateFunction in the singleSpaAngular function.

**Comment by joeldenning** [comment]
Hi @victorlevasseur - yes let's add an `update` lifecycle to single-spa-angular. I think that calling `singleSpaPropsSubject.next()` is the easiest and best way. Thank you for being willing to a submit a PR for this - I'll be happy to review it and get it published once it's merged.

**Comment by joeldenning** [comment]
Resolved in #141
